cookbook-react/recipes/inputs/input-3/styles.scss

```diff
@@ -0,0 +1,83 @@
+$white: #fff;
+
+$input-border: #848486;
+$input-active-color: #0043cd;
+$input-submit: #5a667f;
+$error: #EA4453;
+$disabled: #B5B5B6;
+
+.input-wrapper {
+  height: 40px;
+  margin-bottom: 20px;
+  max-width: 150px;
+  position: relative;
+
+  .input-label {
+    background-color: $white;
+    left: 20%;
```

Comment:
Hmmm be careful with this :thinking: What if you have a form and all fields have a different width? All the labels won't be align. I would choose a fixed value.